central/complianceoperator/v2/pipelines/complianceoperatorinfo/pipeline.go

```diff
@@ -72,7 +75,16 @@ func (s *pipelineImpl) Run(ctx context.Context, clusterID string, msg *central.M
 
 	// if not ready, add it to the status errors
 	if readyPods < desiredPods {
-		operatorErrors = append(operatorErrors, fmt.Sprintf("compliance operator not ready.  Only %d pods are ready when %d are desired.", readyPods, desiredPods))
+		operatorErrors = append(operatorErrors, fmt.Sprintf("compliance operator not ready. Only %d pods are ready when %d are desired.", readyPods, desiredPods))
```

Comment:
```suggestion
		operatorErrors = append(operatorErrors, fmt.Sprintf("compliance operator not ready. Only %d pods are ready whereas %d are desired.", readyPods, desiredPods))
```
Using `when` here makes this have a different meaning :)
We should also avoid `while` as that is used only for time-related aspects.

Reply:
Alternatively `Only %d out of desired %d are ready`. 
There is also one school of tech-writing that discourages using of `Only` as it suggests an evaluation to the sentence and different people may evaluate values differently, for example: "Buy house for only 1M USD" automatically suggests that the price should be considered as good... you know what I mean.

Reply:
I see I ate the word `pods` in my last suggestion alternative. I have a new comment about that, so I resolve this.